src/Compilers/VisualBasic/Test/Semantic/Binding/Binder_Expressions_Tests.vb

```diff
@@ -3041,7 +3037,7 @@ Class Program
             Dim x As Object = ""
             Color.M(x)
         Catch e As Exception
-            Console.WriteLine(e)
+            Console.WriteLine(e.GetType())
```

Comment:
We need to verify that we call VB runtime in a proper way. Simply checking for exception type doesn't do this. I believe that relying on exception message is reasonable in this case, after all we own VB runtime even though is not part of Roslyn's code base. However, alternative ways to verify behavior are welcome. My point is that verification in the current form is probably not sufficient.